### PR TITLE
💵 Money: salary & money bag

### PR DESCRIPTION
**Pull request by Guthen**

### CODE REVIEW

File: gamemodes/virtualroleplay/entities/entities/vrp_money_bag/init.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+function ENT:Initialize()
+    self:SetModel( "models/props/cs_assault/Money.mdl" )
+	self:PhysicsInit( SOLID_VPHYSICS )
+	self:SetMoveType( MOVETYPE_VPHYSICS )
+	self:SetSolid( SOLID_VPHYSICS )
+
+    --  phys
+    local phys = self:GetPhysicsObject()
+	if IsValid( phys ) then
+		phys:Wake()
```
Comment (Erlite) [inline gamemodes/virtualroleplay/entities/entities/vrp_money_bag/init.lua:15]: L'indentation est un peu bizarre ici

**Review by Erlite — APPROVED** [review]
lgtm

File: gamemodes/virtualroleplay/entities/entities/vrp_money_bag/init.lua [hunk 2]
```diff
@@ -0,0 +1,34 @@
+AddCSLuaFile( "cl_init.lua" )
+AddCSLuaFile( "shared.lua" )
+
+include( "shared.lua" )
+
+function ENT:Initialize()
+    self:SetModel( "models/props/cs_assault/Money.mdl" )
```
Comment (Nogitsu) [inline gamemodes/virtualroleplay/entities/entities/vrp_money_bag/init.lua:7]: May look for another model, don't want to need CSS

**Review by Nogitsu — APPROVED** [review]
Working, only aesthetics

**Pull request merged**